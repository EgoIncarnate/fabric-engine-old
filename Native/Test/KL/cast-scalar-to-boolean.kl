function entry()
{
  report "Scalar";
  report (Boolean)0.0;
  report (Boolean)4.5;
}
