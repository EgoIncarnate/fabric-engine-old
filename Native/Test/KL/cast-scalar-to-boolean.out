GlobalList
  Function( 'entry' ('entry'), void )
    ParamList
    CompoundStatement
      StatementList
        Report
          ConstString( 'Scalar' )
        Report
          CastNode( Boolean )
            ConstScalar( 0 )
        Report
          CastNode( Boolean )
            ConstScalar( 4.5 )
Scalar
false
true
